tests/pw/tests/e2e/paypal-marketplace/reconcile.py

```
@@ -0,0 +1,127 @@
+        if ex.get(c) == 'passed':
+            by_area[a][2] += 1
+        if ex.get(c) == 'failed':
+            by_area[a][3] += 1
+
+    print(f'{"area":9} {"cases":>5} {"impl":>5} {"pass":>5} {"fail":>5} {"NOT RUN":>8}')
+    for a in sorted(by_area):
+        c, i, p, f = by_area[a]
+        notrun = i - p - f
+        flag = '  <== never executed' if notrun else ''
+        print(f'{a:9} {c:>5} {i:>5} {p:>5} {f:>5} {notrun:>8}{flag}')
+    tc, ti, tp, tf = (sum(x) for x in zip(*by_area.values()))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)